### PR TITLE
FSPT-119 Remove pyscss

### PR DESCRIPTION
**Pull request by sfount**
We don't take advantage of any of the features of the SCSS pre-processor
so lets remove it and add it back in if we need it.

This is the only use of the `pyscss` filter/ dependency in authenticator so remove it.

There was two files that both seemed to be building assets and one looked unused to remove that.

### CODE REVIEW

File: static_assets.py [hunk 1]
```diff
@@ -29,17 +29,9 @@ def init_assets(app=None, auto_build=False, static_folder="frontend/static/dist"
             output="js/main.min.js",
         )
 
-        css = Bundle(
-            "./scss/*.scss",
-            filters="pyscss,cssmin",
```
Comment (samuelhwilliams) [inline static_assets.py:34]: Can we remove cssmin from dependencies as well?

**Comment by samuelhwilliams** [comment]
Before:
image


After:
image

Looks like this has broken the serving of `main.min.css`? It does only contain one thing (`.fsd-error-message{font-weight:bold;color:#d4351c}`) - but we probably want it still maybe? 🫣

**Comment by sfount** [comment]
I've replaced that one class with a design system one (in the diff) -- thanks for flagging though this PR should remove the reference to the `main.min.css` file

**Comment by samuelhwilliams** [comment]
Ah yes - nice

**Review by samuelhwilliams — APPROVED** [review]
nice tidy up 🙇 🧹

**Pull request merged**